sunpy-core.md

```diff
@@ -6,4 +6,8 @@ Planned release by June 2019 with associated paper. The details of this release
 
 ## 2.0
 
+* Create a generic command to read in files such as `sunpy.open()` or `sunpy.read()`
+* Move to use NDCube as base data object in Maps. NDCube object will become the default object returned if a file is not recognized as a defined base data object.
+* Provide improved spectrum support including fitting data models.
```

Comment:
I think you need to elaborate on this to scope it down a little.

Reply:
Ha! you think?! yes yes yes. good point.

Reply:
astropy models?